tools/generate-test-cases

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python3
+
+import subprocess
+import json
+import tempfile
+import os
+import sys
+
+'''
+This script generates a json test case. It accepts a compose_request as input through standard input.
+
+{
+    "distro": "fedora-30",
+    "arch": "x86_64",
+    "output-format": "qcow2",
+    "filename": "disk.qcow2",
+    "blueprint": {}
+}
+
+It then outputs a json test case as standard output. 
+'''
+
+
+def run_osbuild(manifest, store):
+    osbuild_cmd = ["osbuild", "--store", store, "--json", "-"]
+    result = json.loads(subprocess.check_output(osbuild_cmd, encoding="utf-8", input=json.dumps(manifest)))
+    return result.get("output_id")
+
+
+def get_boot_method(output_format):
+    format_boot_dict = {
+        "partitioned-disk": "nspawn",
+        "ext4-filesystem": "nspawn",
+        "tar": "nspawn-extract",
+        "openstack": "qemu",
+        "qcow2": "qemu",
+        "vhd": "qemu",        
+        "vmdk": "qemu",
+        "ami": "qemu-extract",
+    }
+    return format_boot_dict[output_format]
+
+
+def main(compose_request, store):
```

Comment:
The name `compose_request` can create confusion because of the `store.ComposeRequest` struct we have. We may consider using something like `image_request`.

Reply:
Hm, @larskarlitski do you remember our discussion around this?
If we should rename this, I suggest doing that in a separate PR, as this is just following the naming of the test case structs.

Reply:
Alright, this isn't imo a big issue.